addons/zeus/functions/fnc_moduleGroupSide.sqf

```diff
@@ -25,20 +25,36 @@ if (_side == _newSide) exitWith {};
 private _oldGroup = group _unit;
 private _newGroup = createGroup _newSide;
 
+// Preserve groupid from the previous group if doesn't already exist
```

Comment:
If there is a better way to do this please let me know so I can update it

Reply:
Could use `findIf`

Reply:
From a quick test it looks like this method is faster, and in my opinion more readable than `findIf`.
I'll do a few more tests though since it would make more sense for `findIf` to be faster.